lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -220,6 +252,13 @@ def _save_grade(self, subsection_grade, subsection, course):  # pylint: disable=
         """
         if PersistentGradesEnabledFlag.feature_enabled(course.id):
             subsection_grade.save(self.student, subsection, course)
+        log.warning(u"Persistent Grades: Saved grade for course {0}, version {1}, subtree_edited_on {2}, grade: {3}"
```

Comment:
The `SubsectionGradeFactory` object has the `student`.  So you can add `student` to all its log statements.  Don't need to include it in the single log statement within `SubsectionGrade`.